### PR TITLE
feat: add some terms from numpy and PCL

### PR DESCRIPTION
**Pull request by kminoda**
Address some errors from here:
https://github.com/autowarefoundation/autoware.universe/actions/runs/5251645613/jobs/9486794110?pr=3476

- [np.frombuffer](https://numpy.org/doc/stable/reference/generated/numpy.frombuffer.html)
- [np.tobytes](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tobytes.html)
- [np.allclose](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html)
- [`is_bigendian` in PointCloud2](http://docs.ros.org/en/melodic/api/sensor_msgs/html/msg/PointCloud2.html)

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@kminoda Could you add links to the official documentations or source code?

File: cspell/.cspell.json
Comment (kenji-miyake) [inline cspell/.cspell.json:1]: Please confirm this warning.
image

**Comment by kminoda** [comment]
@kenji-miyake Thanks for the comment, Done!

File: cspell/.cspell.json
Comment (kminoda) [reply cspell/.cspell.json:1]: https://github.com/tier4/autoware-spell-check-dict/pull/540/commits/2b898594981a481e90742beffecbde7cb8b6e552
Addressed here

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**